packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/string_database/index_db/indexed_db_adapter.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'dart:async';
+import 'dart:js_util';
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/string_database/string_database.dart';
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/string_database/string_database_element.dart';
+import 'package:amplify_core/amplify_core.dart';
+// ignore: implementation_imports
+import 'package:aws_common/src/js/indexed_db.dart';
```

Comment:
Should `indexed_db` be exported from `aws_common/aws_common.dart`?

Reply:
I'm not sure ... in the aws_common package none of the js subfiles are exported.  

SecureStorage which uses `indexed_db` also does the same thing. 

I'm not 100% sure why but I can imagine it's because the files in `js` are more private/internal.  While the other types in `aws_common` could feasibly be used by outside developers in a more generic use case.  

@Jordan-Nelson does that reasoning seem correct to you?

Reply:
Shouldn't be exported since they are experimental and will likely be replaced by https://github.com/dart-lang/web/ in the not-too-distant future.